src/main/java/gov/cabinetoffice/gap/applybackend/validation/validators/QuestionResponseValidator.java

```diff
@@ -102,46 +102,48 @@ private ValidationResult validate(CreateQuestionResponseDto submittedQuestion, S
             }
         }
 
-        if (validation.getMinLength() != null &&
-                submittedQuestion.getResponse().length() < validation.getMinLength()) {
-            result.addError(ValidationConstants.SINGLE_RESPONSE_FIELD, String.format("Answer must be %s characters or more", validation.getMinLength()));
-            return result;
-        }
+        if (!singleResponseIsEmpty) {
```

Comment:
this is kind of hard to read from the diff but this change is just moving all the single response validation logic inside this condition, so it can't accidentally fire on a multi-response when the object contains unexpected values